torch_xla/csrc/helpers.h

```diff
@@ -324,8 +324,14 @@ class XlaHelpers {
 
   template <typename T>
   static xla::Literal Range(T start, T end, T step) {
+    int64_t range_size = (int64_t)std::ceil((end - start) / step);
+    xla::Array<T> range({range_size});
+    range.FillWithMultiples(step);
+    range.Each([start](absl::Span<const int64_t> index, T* value) {
```

Comment:
micro-nit: can you capture `start` by reference?

From our style guide: [`Use default capture by reference ([&]) only when the lifetime of the lambda is obviously shorter than any potential captures.`](https://google.github.io/styleguide/cppguide.html#Lambda_expressions)